src/LAMMPS.jl

```diff
@@ -233,6 +241,51 @@ function command(lmp::LMP, cmd::Union{String, Array{String}})
     check(lmp)
 end
 
+"""
+    create_atoms(
+        lmp::LMP, x::Matrix{Float64}, id::Vector{Int32}, types::Vector{Int32};
+        v::Union{Ptr{Float64},Matrix{Float64}}=Ptr{Float64}(C_NULL),
+        image::Union{Ptr{IMAGEINT},Vector{IMAGEINT}}=Ptr{IMAGEINT}(C_NULL),
+        bexpand::Bool=false
+    )
+
+Create atoms for a LAMMPS instance. 
+x contains the atom positions and should be a 3 by Matrix{Float64}, where n is the number of atoms. 
+id contains the id of each atom and should all be Vector{Int32} with length n.
+types contains the atomic type (LAMMPS number) of each atom and should all be a Vector{Int32} with length n.
+v contains the associated velocities and should be a 3 by Matrix{Float64}.
+image contains the image flags for each atom and should be Vector{IMAGEINT} with length n.
+bexpand is a Bool that defines whether or not the box should be expanded to fit the input atoms (default not).
+"""
+function create_atoms(
+    lmp::LMP, x::Matrix{Float64}, id::Vector{Int32}, types::Vector{Int32};
+    v::Union{Nothing,Matrix{Float64}}=nothing,
+    image::Union{Nothing,Vector{IMAGEINT}}=nothing,
+    bexpand::Bool=false
+)
+    numAtoms = size(x, 2)
+    if size(x, 1) != 3
+        throw(ArgumentError("x must be a n by 3 matrix, where n is the number of atoms"))
+    end
+    if numAtoms != length(id)
+        throw(ArgumentError("id must have the same length as the number of atoms"))
+    end
+    if numAtoms != length(types)
+        throw(ArgumentError("types must have the same length as the number of atoms"))
+    end
+    if v != nothing && size(x) != size(v)
+        throw(ArgumentError("x and v must be the same size"))
+    end
+    if image != nothing && numAtoms != length(image)
+        throw(ArgumentError("image must have the same length as the number of atoms"))
+    end
+
+    v = v == nothing ? Ptr{Float64}(C_NULL) : v
+    image = image == nothing ? Ptr{IMAGEINT}(C_NULL) : image
```

Comment:
```suggestion
    v = v == nothing ? C_NULL : v
    image = image == nothing ? C_NULL : image
```
`Base.unsafe_convert` can handle the type conversion